### ISSUE TITLE
More for the wishlist: Folders

### ISSUE THREAD

**Issue by chrisschrader** [umfana/FaviconBar#20]
Hello, first of - thanks for the extension. Helps me a lot to find my way around the bookmarks. 
But I can only second some ideas put forward here: adding new icons for addresses that need them, easier reordering, mix text and icons, and finally my own addition: folders.
Safari lets me put folders into the bookmark bar, to organize addresses under topics. Switching from the regular bar to FaviconBar I am really missing that. Any hope you'll include that? Best, Chris

**Comment by popcornhax** [comment]
From my basic understanding of Safari Extensions folders might be difficult, but I have a couple ideas of how they could be pulled off.

**Comment by popcornhax** [comment]
For folders: I did not get this in release 1.3 but I am still working this issue for future releases (the foundation to it is built, but it has proven to be difficult)

The other features mentioned above included in the current release.

**Comment by SJCman** [comment]
Yes please put me down for FOLDERS too!  I use them on the favicon bar on both Chrome and Firefox.  In FF they just showed up automatically when I brought in a big list of favorites into what they call the "Bookmarks Toolbar".  

Anyway, despite the missing folders, your favicon bar for Safari is Awesome!

**Comment by popcornhax** [comment]
SJCman:  Thanks for the comments.  As for folders, it's planned but it might not make everybody happy.  There are only two ways I can figure out to do folders.  The first is a drop-down text menu with link names but no icons, and the second would be a horizontal expanding menu with icons.  I'm trying both, we'll see what works and what doesn't.  Sadly I have no way to create a drop-down menu with icons in Safari's extension framework :(

**Comment by moshed** [comment]
im on board with horizontally expanding folders that sounds great!
